api/leaderboard/models.py

```
@@ -291,7 +292,7 @@ class UnifiedScoreHistory(models.Model):
  
     class Meta:
         # one record per user per day
-        unique_together = ("username", "date")
+        unique_together = ("user_id", "date")
         ordering = ["date"]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)